test/utils/blueprints/layout_blueprints.hpp

```diff
@@ -985,6 +987,7 @@ Lyt bestagon_crossing() noexcept
 {
     static_assert(fiction::is_cell_level_layout_v<Lyt>, "Lyt is not a cell-level layout");
     static_assert(fiction::has_sidb_technology_v<Lyt>, "Lyt is not an SiDB layout");
+    static_assert(fiction::has_siqad_coord_v<Lyt>, "Lyt is not an SiDB layout");
```

Comment:
The string doesn't match the check. This is true for all `has_siqad_coord_v` checks added to this file.